core/src/stories/patterns/navigation/navigation-fewitems.stories.tsx

```diff
@@ -183,7 +183,8 @@ const Template = () =>
           </tds-header-dropdown-list>
         </tds-header-dropdown>
 
-        <tds-header-brand-symbol slot="end" href="https://design.scania.com" aria-label="Scania - red gryphon on blue shield">
+        <tds-header-brand-symbol slot="end" aria-label="Scania - red gryphon on blue shield">
```

Comment:
The aria label should be moved to the `<a>` tag since that is the element with the image.